yaml/roads.yaml

```diff
@@ -32,6 +31,14 @@ global:
       sport: {col: sport}
       man_made: {col: man_made}
       walking_network: {expr: "mz_hiking_network(osm_id)"}
+      bicycle_network: {expr: "mz_cycling_network(tags, osm_id)"}
+      oneway: {col: oneway}
```

Comment:
What do we do in the situation where both exist? If they both say reverse, just reverse the geometry once? And otherwise, would we use the `oneway` tag's reversal orientation be what we honor first, and make the `bicycle:oneway` direction match accordingly? (If this doesn't make sense I can enumerate the cases)